libraries/bxdf/lama/lama_dielectric.mtlx

```diff
@@ -0,0 +1,167 @@
+      <input name="amount" type="float" nodename="tangent_rotate_degree_offset" />
+      <input name="axis" type="vector3" interfacename="normal" />
+    </rotate3d>
+    <normalize name="tangent_rotate_normalize" type="vector3">
+      <input name="in" type="vector3" nodename="tangent_rotate" />
+    </normalize>
+
+    <!-- Interior -->
+    <divide name="absorption" type="color3">
+      <input name="in1" type="color3" interfacename="absorptionColor" />
+      <input name="in2" type="float" interfacename="absorptionRadius" />
+    </divide>
+    <convert name="absorption_vector" type="vector3">
+      <input name="in" type="color3" nodename="absorption" />
+    </convert>
+    <convert name="scatter_vector" type="vector3">
+      <input name="in" type="color3" interfacename="scatterColor" />
+    </convert>
+    <anisotropic_vdf name="interior_vdf" type="VDF">
+      <input name="absorption" type="vector3" nodename="absorption_vector" />
+      <input name="scattering" type="vector3" nodename="scatter_vector" />
+      <input name="anisotropy" type="float" interfacename="scatterAnisotropy" />
+    </anisotropic_vdf>
+
+    <!-- BTDF -->
+    <dielectric_bsdf name="transmission_bsdf" type="BSDF">
+      <input name="weight" type="float" value="1.0" />
+      <input name="tint" type="color3" interfacename="transmissionTint" />
+      <input name="ior" type="float" nodename="relative_ior" />
+      <input name="roughness" type="vector2" nodename="roughness_anisotropic_squared_clamped" />
+      <input name="normal" type="vector3" interfacename="normal" />
+      <input name="tangent" type="vector3" nodename="tangent_rotate_normalize" />
+      <input name="distribution" type="string" value="ggx" />
+      <input name="scatter_mode" type="string" value="T" />
+    </dielectric_bsdf>
+    <layer name="transmission_layer" type="BSDF">
+      <input name="top" type="BSDF" nodename="transmission_bsdf" />
+      <!-- TODO: uncomment once layering BSDFs over VDFs is supported.
+      <input name="base" type="VDF" nodename="interior_vdf" />
+      -->
+    </layer>
+
+    <!-- BRDF -->
+    <dielectric_bsdf name="reflection_bsdf" type="BSDF">
+      <input name="weight" type="float" value="1.0" />
+      <input name="tint" type="color3" interfacename="reflectionTint" />
+      <input name="ior" type="float" nodename="relative_ior" />
+      <input name="roughness" type="vector2" nodename="roughness_anisotropic_squared_clamped" />
+      <input name="normal" type="vector3" interfacename="normal" />
+      <input name="tangent" type="vector3" nodename="tangent_rotate_normalize" />
+      <input name="distribution" type="string" value="ggx" />
+      <input name="scatter_mode" type="string" value="R" />
+    </dielectric_bsdf>
+
+    <!-- BSDF -->
+    <add name="dielectric_bsdf" type="BSDF">
+      <input name="in1" type="BSDF" nodename="reflection_bsdf" />
+      <input name="in2" type="BSDF" nodename="transmission_layer" />
+    </add>
```

Comment:
To my mind it would be more accurate to use a BSDF `layer` node than a BSDF `add` node here. To make sure the base is attenuated according to the top component's reflectance.

Using an `add` node implies that the base transmission BSDF must include an inverse Fresnel factor. This is not the case for the implementation of dielectric_bsdf in "T" scatter mode (afaik).